core/streamdata.go

```diff
@@ -30,6 +30,8 @@ type DetectionConfig struct {
 
 type StreamParameters struct {
 	ManifestID   ManifestID
+	RealStreamID string // lol
```

Comment:
Need suggestions about how to properly name this field. We already have a `StreamID` method below for implementing the `AppData` interface, so it can't be called that. I didn't want to make just an aesthetic difference like `StreamID_` either, since they actually do not refer to the same values. Unless any of you can assure me I can simply return the `StreamID_` from the auth webhook from the `StreamID` function below.